Sources/NIOHTTP2/HTTP2StreamMultiplexer.swift

```diff
@@ -282,13 +282,18 @@ extension HTTP2StreamMultiplexer {
 extension HTTP2StreamMultiplexer {
     /// HTTP2StreamMultiplexer.SendableView exposes only the thread-safe API of HTTP2StreamMultiplexer
     ///
-    /// We use unckecked Sendable here because we always make sure we are on the right event loop
+    /// We use unchecked Sendable here because we always make sure we are on the right event loop
     /// from this code on.
-    struct SendableView: @unchecked Sendable {
+    public struct SendableView: @unchecked Sendable {
         let http2StreamMultiplexer: HTTP2StreamMultiplexer
         let eventLoop: EventLoop
 
-        func createStreamChannel(promise: EventLoopPromise<Channel>?, _ streamStateInitializer: @escaping NIOChannelInitializer) {
+        public init(http2StreamMultiplexer: HTTP2StreamMultiplexer, eventLoop: EventLoop) {
```

Comment:
Do we want the `init` to be `public`? Normally the sendable view would be created by the `HTTP2StreamMultuplexer`.